Detectors/MUON/MID/Tracking/src/Tracker.cxx

```diff
@@ -396,26 +387,26 @@ void Tracker::finalizeTrack(Track& track)
   }
   track.setChi2(chi2);
   track.setNDF(ndf);
-  LOG(DEBUG) << track;
+  LOG(DEBUG) << "Finalize track: " << track;
 }
 
 //______________________________________________________________________________
 bool Tracker::addTrack(const Track& track)
```

Comment:
Consider calling this one tryAddTrack as it more closely describe what's going on ?

Reply:
Done